agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2187,9 +2187,13 @@ bool backhaul_manager::handle_slave_backhaul_message(std::shared_ptr<sRadioInfo>
         break;
     }
     default: {
-        LOG(ERROR) << "Unhandled message received from master: "
-                   << int(beerocks_header->action_op());
-        return false;
+        bool handled = m_task_pool.handle_cmdu(cmdu_rx, sMacAddr(), beerocks_header);
```

Comment:
nitpick:
I think instead of handling each type separately (1905/backhaul/etc..) there should be one call from the `backhaul_manager::handle_cmdu` main method (the one that calls the different types of handling.
I understand this required refactoring the main method and you didn't want to get into this for this PR - but consider this change in the coming PRs